File: modul 6/python/PRAK605-2410817110008-MuhammadAzmaAlFaqih.py
ordo = int(input())
print("Matriks A")
matriksA = []
matriksB = []
matriksAxB = []
while len(matriksA)<ordo:
    inputs = list(map(int, input().split()))
    matriksA.append(inputs)
print("Matriks B")
while len(matriksB)<ordo:
    inputs = list(map(int, input().split()))
    matriksB.append(inputs)
print("Matriks AxB")
for i in range(ordo):
    matriksAxB.append([])
    for j in range(ordo):
        nilai = 0
        for k in range(ordo):
            nilai += matriksA[i][k] * matriksB[k][j]
        matriksAxB[i].append(nilai)
for i in range(ordo):
    for j in range(ordo):
        print(matriksAxB[i][j], end=' ')
    print()

File: modul 6/c/PRAK605-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int main(){
  int ordo;
  scanf("%d", &ordo);
  int matriksA[ordo][ordo];
  int matriksB[ordo][ordo];
  int hasil[ordo][ordo];
  printf("Matriks A\n");
  for(int i = 0; i<ordo; i++){
    for(int j = 0; j<ordo; j++){
      scanf("%d", &matriksA[i][j]);
    }
  }
  printf("Matriks B\n");
  for(int i = 0; i<ordo; i++){
    for(int j = 0; j<ordo; j++){
      scanf("%d", &matriksB[i][j]);
    }
  }
  printf("Matriks AxB\n");
  for(int i = 0; i<ordo; i++){
    for(int j = 0; j<ordo; j++){
      int jumlah = 0;
      for(int k = 0; k<ordo; k++){
        jumlah += matriksA[i][k] * matriksB[k][j];
      }
      hasil[i][j] = jumlah;
    }
  }

  for(int i = 0; i<ordo; i++){
    for(int j = 0; j<ordo; j++){
      printf("%d ", hasil[i][j]);
    }
    printf("\n");
  }
}